code/game/turfs/walls/wall_types.dm

```diff
@@ -273,11 +273,24 @@
 //	icon_state = "title_holiday"
 	layer = FLY_LAYER
 	pixel_x = -64
+	//random title as starting point so it isnt choking every time you see it.
+	var/list/total_titles
+	///Gets the current title number. It sets itself with the proc below, no need to touch.
+	var/icon/current_title
```

Comment:
What? You're storing a string in an icon type var, why?

Reply:
It didn't work like that.

Reply:
Icon states are strings, not icons